### PR TITLE
Update LootHelper.cs

### PR DESCRIPTION
**Pull request by ouaklafoud**
Log into the main window what is the loot item, and if it has been looted.
For player knowledge but mainly for famebot users

### CODE REVIEW

**Comment by thomas-crane** [comment]
I won't get a chance to test this until later today, but the code looks fine so I should be able to merge this after testing.

**Comment by ouaklafoud** [comment]
This let me know i left a dirk of cronus on the floor .... so it worth the look 🥇 
maybe you ll find out how to better state the item name, but for me it seems to work fine.. from time to time i have this error in the NEWTICK, UPDATE package

**Comment by ouaklafoud** [comment]
I commented here and there the PluginUtils.Log of "Client received" , "Connected to remote host." and "Disconnected.", then I get no error displayed anymore ever from loot logs I added

**Comment by ouaklafoud** [comment]
allows to load from external packets.xml

**Comment by thomas-crane** [comment]
My apologies, I haven't had a chance to test this yet since I'm on holiday at the moment and only have access to a Mac. I've looked through the code and can't see any issues which might come up from testing, so I will probably just merge it anyway :smile: I can always revert it if there is an issue anyway.

File: Lib K Relay/GameData/GameData.cs [hunk 1]
```diff
@@ -113,8 +113,16 @@ static GameData()
 	            PluginUtils.Log("GameData", "Mapped {0} objects.", Objects.Map.Count);
             },
             () => {
-                Packets = new GameDataMap<byte, PacketStructure>(PacketStructure.Load(XDocument.Parse(RawPacketsXML)));
-	            PluginUtils.Log("GameData", "Mapped {0} packets.", Packets.Map.Count);
+                 try
+                {
+                    Packets = new GameDataMap<byte, PacketStructure>(PacketStructure.Load(XDocument.Load("Packets.xml")));
+                    PluginUtils.Log("GameData", "loaded from file!");
```
Comment (thomas-crane) [inline Lib K Relay/GameData/GameData.cs:119]: Can this be a bit more descriptive, e.g. `Loaded packets from Packets.xml`?

Other than that, these changes look fine. I will merge them after this change

**Review by thomas-crane — CHANGES_REQUESTED** [review]

**Pull request merged**